src/components/MediaPlayer/VideoJS/videojs-theme.scss

```diff
@@ -0,0 +1,178 @@
+/*
+  Adapted from: https://github.com/videojs/themes/blob/master/city/index.css
+*/
+
+/* Minimum height and width for audio player */
+.vjs-theme-ramp.vjs-audio-only-mode {
+  min-width: 490px;
+  min-height: 50px;
```

Comment:
I'm concerned this may cause problems with existing audio embeds since they only have 40px.

https://github.com/avalonmediasystem/avalon/blob/37cfbea682640796d9d33f4069c8a773537f581e/app/models/concerns/master_file_behavior.rb#L20

Reply:
Yes, it will probably get cut-off. Let me see whether this can be changed.

Reply:
Good catch! 🚀 I changed the `min-height` for audio player to `40px`. It still works with this value as expected.